spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -32,11 +32,15 @@
 @ConfigurationProperties("spring.cloud.gcp.pubsub")
 public class GcpPubSubProperties implements CredentialsSupplier {
 
+	// Contains settings specific to the subscriber factory
 	private final Subscriber subscriber = new Subscriber();
 
+	// Contains settings specific to the publisher factory
```

Comment:
That's not JavaDocs comment...

Reply:
I'm really confused. I thought he said there "must not" be one. 

So I added a comment for your benefit and for any reader who is wondering what this is.

Reply:
OK. As you wish. But what I believe @snicoll meant exactly `do not have to`, not `must not`: https://www.gymglish.com/en/english-grammar/must-vs-have-to. And his point that this JavaDoc on such a complex property won't be visible in the meta-data, just because it isn't a simple linear property, but the entire namespace.

P.S. Pay attention to the first paragraph in that "must-vs-have-to" article. That's why I used `must` there - my opinion, not a rule.

(Sorry for the spam though: English is not my native language :smile: )

Reply:
Thanks for the link.